Alcolator/Alcolator/AlcoholicBeverage.h

```diff
@@ -0,0 +1,21 @@
+//
+//  AlcoholicBeverage.h
+//  Alcolator
+//
+//  Created by Ryan Walker on 9/30/15.
+//  Copyright © 2015 Ryan Walker. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface AlcoholicBeverage : NSObject
+
+@property (strong, nonatomic) NSString *name;
+@property float alcoholicPercentage;
+@property float typicalSizeOfServing;
+
```

Comment:
The properties above are really well-named, I might just name the methods below so that their purpose (and inputs/outputs) are that much more clear. Like so:
```
- (float)ouncesOfAlcoholPerServing; // Perfect.
- (float)alcoholServingsPerOunces:(float)ouncesOfAlcohol; // This might be a little more clear about what we're returning...
//- (float)convertAlcoholicServings:(float)ouncesOfAlcohol;
- (float)ouncesOfAlcoholTotalForNumberOfBeverages:(float)numberOfBeverages; // This just makes it clear what we're passing in... you could argue that the parameter already does this, it's just common practice in iOS and Mac OS X to make the methods pretty descriptive.
//- (float)ouncesOfAlcoholTotal:(float)numberOfBeverages;
```

Again, these are more grammatical choices than anything wrong. Overall these methods are great and consider the problems we're trying to solve.